report-service/src/main/java/gov/cdc/dataingestion/nbs/converters/Hl7ToRhapsodysXmlConverter.java

```diff
@@ -171,26 +169,11 @@ private Hl7ToRhapsodysXmlConverter() {
     }
 
     public String convert(String hl7Msg) throws Exception {
-
         String rhapsodyXml = "";
 
         HL7Helper hl7Helper = new HL7Helper();
-
-
         HL7ParsedMessage hl7ParsedMsg = hl7Helper.hl7StringParser(hl7Msg);
 
-        // this should be handled by validation pipeline
-        // the one scenario this might be needed is when someone reprocessing the failed message on this pipeline
-        switch (hl7ParsedMsg.getOriginalVersion()) {
```

Comment:
could you leave this as it is for now, also ignore the comment above the code. 
This basically convert 231 to 251 if applicable.

I will move this into library function

Reply:
Will fix in next release after api's get updated.